### PR TITLE
WRN-4396: Migrate Checkbox, CheckboxItem, RadioItem, Switch, SwitchItem unit tests to Testing Library.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [X] Documentation was verified or is not changed
* [X] UI test was passed or is not needed
* [X] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated tests from Enzyme to Testing Library.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Changed test structure to work with Testing Library.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-4396

### CODE REVIEW

File: Checkbox/tests/Checkbox-specs.js [hunk 1]
```diff
@@ -1,63 +1,52 @@
...
 
-		expect(actual).toContain(expected);
+		expect(checkbox).toHaveClass(expected);
 	});
 
 	test('should prioritize indeterminate over selected', () => {
-		const checkbox = shallow(
-			<CheckboxBase indeterminate selected indeterminateIcon="minus">Sel</CheckboxBase>
-		);
+		render(<CheckboxBase indeterminate selected indeterminateIcon="minus">Sel</CheckboxBase>);
```
Comment (stanca-pop-lgp) [inline Checkbox/tests/Checkbox-specs.js:45]: Please sort props.

File: Checkbox/tests/Checkbox-specs.js [hunk 2]
```diff
@@ -1,63 +1,52 @@
...
-
-		const expected = 1;
-		const actual =  checkbox.find(`.${css.selected}`).length;
+		const expected = '✓';
 
-		expect(actual).toBe(expected);
+		expect(checkbox).toHaveTextContent(expected);
 	});
 
 	// Extra test cases
```
Comment (stanca-pop-lgp) [inline Checkbox/tests/Checkbox-specs.js:25]: You can remove this comment.

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

File: CheckboxItem/tests/CheckboxItem-specs.js [hunk 3]
```diff
@@ -1,56 +1,42 @@
...
-
-		const subject = shallow(
-			<CheckboxItemBase icon={expected}>
-				Hello CheckboxItem
-			</CheckboxItemBase>
-		);
+		render(<CheckboxItemBase icon="star">Hello CheckboxItem</CheckboxItemBase>);
 
-		const actual = subject.find('Checkbox').prop('children');
+		const expected = 983272;
```
Comment (adrian-cocoara-lgp) [inline CheckboxItem/tests/CheckboxItem-specs.js:10]: Leave a comment here: // decimal converted charCode of Unicode 'star' character

File: RadioItem/tests/RadioItem-specs.js [hunk 4]
```diff
@@ -1,71 +1,52 @@
...
-		const radioItem = mount(
-			<RadioItem>
-				Hello RadioItem
-			</RadioItem>
-		);
+		render(<RadioItem>Hello RadioItem</RadioItem>);
 
-		const expected = 0;
-		const actual = radioItem.find(`.${css.selected}`).length;
+		const expected = 983071;
```
Comment (adrian-cocoara-lgp) [inline RadioItem/tests/RadioItem-specs.js:20]: Leave a comment here: // decimal converted charCode of Unicode 'iconName' character

File: RadioItem/tests/RadioItem-specs.js [hunk 5]
```diff
@@ -1,71 +1,52 @@
...
-		const radioItem = mount(
-			<RadioItem selected>
-				Hello RadioItem
-			</RadioItem>
-		);
+		render(<RadioItem selected>Hello RadioItem</RadioItem>);
 
-		const expected = 1;
-		const actual = radioItem.find(`.${css.selected}`).first().length;
+		const expected = 983071;
```
Comment (adrian-cocoara-lgp) [inline RadioItem/tests/RadioItem-specs.js:29]: Leave a comment here: // decimal converted charCode of Unicode 'iconName' character

**Pull request merged**